app/assets/javascripts/modules/FirmMap.js

```diff
@@ -130,12 +130,15 @@ define(['jquery', 'DoughBaseComponent'],
    * @returns {Object} Config for creating a google maps marker with
    */
   FirmMapProto._generateMarkerConfig = function($element) {
+    var pinUrlString = $element.data('dough-map-point-type') + 'PinUrl';
+    var iconUrl = eval('this.config.' + pinUrlString);
```

Comment:
LOL ok travis doesn't like it at all :)

![screen shot 2015-11-11 at 11 54 27](https://cloud.githubusercontent.com/assets/306583/11090603/12f92f76-886b-11e5-83b6-5d425ce084ea.png)